views/article.html

```diff
@@ -67,7 +71,7 @@
 		</script>
 	</head>
 	<body>
-		<amp-analytics config="//SOURCE_HOST{{SOURCE_PORT}}/analytics/config.json">
+		<amp-analytics config="//SOURCE_HOSTNAME{{SOURCE_PORT}}/analytics/config.json">
```

Comment:
I fixed this in the ads branch by removing `SOURCE_HOST` entirely and just using a relative path